gce/scripts/startup-script.sh

```diff
@@ -19,26 +19,31 @@ set -v
 
 # Talk to the metadata server to get the project id
 PROJECTID=$(curl -s "http://metadata.google.internal/computeMetadata/v1/project/project-id" -H "Metadata-Flavor: Google")
-BUCKET=$(curl -s "http://metadata.google.internal/computeMetadata/v1/instance/attributes/BUCKET" -H "Metadata-Flavor: Google")
 
-echo "Project ID: ${PROJECTID}  Bucket: ${BUCKET}"
-
-# get our file(s)
-gsutil cp "gs://${BUCKET}/gce/"** .
+echo "Project ID: ${PROJECTID}"
 
 # Install dependencies from apt
-apt-get update
-apt-get install -yq openjdk-11-jre-headless
+apt-get install -yq openjdk-8-jdk git
+java -version
+
+# Install updated version of Maven
+wget https://www-us.apache.org/dist/maven/maven-3/3.6.2/binaries/apache-maven-3.6.2-bin.tar.gz -P /tmp
```

Comment:
Can we install this with `apt` instead?

Reply:
Let me confirm that the version available through apt (3.6.0) works when run in GCE. There were a lot of components that needed to be tweaked to get this to run.